MdeModulePkg/Library/TraceHubDebugSysTLib/BaseTraceHubDebugSysTLib.inf

```diff
@@ -42,3 +42,6 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdTraceHubDebugLevel
   gEfiMdeModulePkgTokenSpaceGuid.PcdEnableTraceHubDebugMsg
   gEfiMdeModulePkgTokenSpaceGuid.PcdTraceHubDebugMmioAddress
+
+[BuildOptions]
+  *_*_*_CC_FLAGS  = -I$(WORKSPACE)/MdePkg/Library/MipiSysTLib/mipisyst/library/include

```

Comment:
Are headers in mipisyst/library/include needed when building libraries that depend on MipiSysTLib?  I would think they'd only be needed when building MipiSysTLib itself.  And consuming libraries would get what they need from a MipiSysTLib.h.

Reply:
CryptoPkg.dec contains:
```
[Includes]
  Include

[Includes.Common.Private]
  Private
  Library/Include
  Library/OpensslLib/openssl/include
```

And RedfishPkg.dec contains:
```
[Includes.Common.Private]
  PrivateInclude/Crt            # Private header files for C RTL.
  Library/JsonLib               # Private header files for jansson
                                # configuration files.
                                #  - jansson_config.h
                                #  - jansson_private_config.h
                                # jansson.h refers to above two configuration
                                # files for building platform jansson library.
  Library/JsonLib/jansson/src   # For referring to jannson.h
```

So, there's precedent for adding 3rdparty library includes in the dec file.  However, as `Includes.Common.Private`.  I'm guessing the `Private` means modules in the package get the include path, but not consumers of the package.

Unfortunately, even `[Includes.Common.Private]` would still require developers to populate the mipisyst directory.  In the case of CryptoPkg and openssl, that's not an issue because it's hard to use CryptoPkg without openssl.  But mipisyst isn't as essential to MdeModulePkg.